src/core_modules/capture-core/components/FormFields/New/HOC/withLabel.js

```diff
@@ -16,8 +17,11 @@ const getStyles = (theme: Theme) => ({
 
 const getStylesLabel = (theme: Theme) => ({
     container: {
-        display: 'flex',
+        display: 'block',
```

Comment:
`div` elements have default display value `block`. You might wanna remove this line completely if it doesnt serve any other reason. Does it?

Reply:
Thanks! It does not serve any other reason, so i have updated with another commit.